README.md

```diff
@@ -132,6 +132,29 @@ docker run --rm -it -v $(pwd):/tmp -w /tmp objectstore bash
 Refer to the rest of this document for instructions to build/test in other
 configurations.
 
+## Running [app] tests against a local MongoDB Stitch
+
+Stitch images are published to our private Github CI. Follow the steps here to
+set up authorization from docker to your Github account https://github.com/realm/ci/packages/147854
+Once authorized, run the following docker commands from the top directory to start a local instance:
+
+```
+docker build --rm --tag local-stitch -f workflow/stitch.Dockerfile workflow
+docker run -p 9090:9090 -it -v $(pwd)/tests/mongodb:/app_config:rw local-stitch /bin/bash -c "(import_app.sh&) && run.sh"
```

Comment:
```suggestion
docker build --tag local-stitch -f workflow/stitch.Dockerfile workflow
docker run --rm -p 9090:9090 -v $(pwd)/tests/mongodb:/app_config:rw -it local-stitch /bin/bash -c "(import_app.sh&) && run.sh"
```
`--rm` should be applied on `docker run`, and the order of `-it` is significant in that it usually always needs to immediately precede the image name.